frontend/packages/core/src/Table/table.tsx

```diff
@@ -132,12 +132,13 @@ const TableRow = ({
   cellDefault,
   responsive = false,
   colSpan,
+  style = [],
   ...props
 }: TableRowProps) => (
   <StyledTableRow onClick={onClick} $responsive={responsive} {...props}>
     {React.Children.map(children, (value, index) => (
       // eslint-disable-next-line react/no-array-index-key
-      <TableCell key={index} responsive={responsive} colSpan={colSpan}>
+      <TableCell key={index} responsive={responsive} colSpan={colSpan} style={style?.[index]}>
```

Comment:
Hmm, I'm not sure I'm a huge fan of this. It seems finicky if we only want to style one or two cells and not the entire thing.

I wonder if in the child map if we can just detect if we're already a `TableCell` and then just pass it through without nesting it like it currently does. That would allow us to make a styled cell in the parent component and then the row doesn't even have to know about it.